audit/projects/k8s-staging-capi-docker/services/dns/info.json

```diff
@@ -1,53 +0,0 @@
-{
-  "id": "k8s-staging-capi-docker",
-  "kind": "dns#project",
```

Comment:
Removal of dns info is expected due to disablement of the dns service (see other review comments); this service should not have been enabled in the first place